lib/grizzly/zwave/command_classes/command_class_version.ex

```diff
@@ -1,6 +1,6 @@
-defmodule Grizzly.ZWave.CommandClasses.Version do
+defmodule Grizzly.ZWave.CommandClasses.CommandClassVersion do
```

Comment:
The Z-Wave command class that contains the command class get and report commands is `version`.

Reply:
Oops. I went by the XML document where it is call COMMAND_CLASS_VERSION. Making changes back to VERSION.